src/utils/options.ts

```diff
@@ -302,6 +302,11 @@ const order = new commander.Option(
   .choices(['asc', 'desc'])
   .default('asc');
 
+const recursive = new commander.Option(
+  '-r --recursive',
+  'If enabled, specified directories will be removed along with their contents',
+).default(false);
+
```

Comment:
I don't think that should be done for this command. Deleting a directory is impossible without the `--recursive` flag det, and deleting directories is a common enough operation that it would be a major annoyance for the users. 

Moreover, it would conflict with programmers' muscle memory, defeating the purpose of UNIX command parity that we are trying to implement.

Reply:
You misunderstand me. I just mean remove the `-r` so it's only `--recursive`. I'd like to have the short version but we've found that it's too likely to conflict with the higher level command options.

Reply:
Oh sorry, I also wasn't sufficiently clear. I meant that deleting directories is a common enough operation that typing `--recursive` each time we want to do that seems like a pain.

On that note, I don't think we even can implement the -f option, as that option forcefully removes unwritable files, but the vaults don't have any concept of actual file perms, so it will be redundant.

Reply:
See: https://github.com/MatrixAI/Polykey-CLI/issues/278#issuecomment-2342559234 for the problem.